From 503f48a0a43ddf20098b2a5ec2c3d9d91775e441 Mon Sep 17 00:00:00 2001
From: Alison Wang <b18965@freescale.com>
Date: Wed, 9 Dec 2015 10:53:04 +0800
Subject: [PATCH 04/70] armv8: aarch32: Add ITS file for AArch32 Linux on
 LS1043ARDB

kernel-ls1043a-rdb-aarch32.its is added to load kernel, DTB
and root filesystrem together.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
---
 kernel-ls1043a-rdb-aarch32.its |   53 ++++++++++++++++++++++++++++++++++++++++
 1 file changed, 53 insertions(+)
 create mode 100644 kernel-ls1043a-rdb-aarch32.its

--- /dev/null
+++ b/kernel-ls1043a-rdb-aarch32.its
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2015, Freescale Semiconductor
+ *
+ * This file is licensed under the terms of the GNU General Public
+ * License version 2.  This program is licensed "as is" without any
+ * warranty of any kind, whether express or implied.
+ */
+
+/dts-v1/;
+
+/ {
+	description = "RDB Image file for the LS1043A Linux Kernel";
+	#address-cells = <1>;
+
+	images {
+		kernel@1 {
+			description = "ARM32 Linux kernel";
+			data = /incbin/("./arch/arm/boot/zImage");
+			type = "kernel";
+			arch = "arm";
+			os = "linux";
+			compression = "none";
+			load = <0x80008000>;
+			entry = <0x80008000>;
+		};
+		fdt@1 {
+			description = "Flattened Device Tree blob";
+			data = /incbin/("./fsl-ls1043a-rdb.dtb");
+			type = "flat_dt";
+			arch = "arm";
+			compression = "none";
+			load = <0x90000000>;
+		};
+		ramdisk@1 {
+			description = "LS1 Ramdisk";
+                        data = /incbin/("./fsl-image-core-ls1021atwr-wifi.rootfs.ext2.gz");
+			type = "ramdisk";
+			arch = "arm";
+			os = "linux";
+			compression = "none";
+		};
+	};
+
+	configurations {
+		default = "config@1";
+		config@1 {
+			description = "Boot Linux kernel";
+			kernel = "kernel@1";
+			fdt = "fdt@1";
+			ramdisk = "ramdisk@1";
+		};
+	};
+};
